packages/webpack-utils/src/fabricAsyncLoader.ts

```diff
@@ -0,0 +1,23 @@
+// This code is heavily inspired by https://github.com/baflo/react-loadable-loader.
+import * as webpack from 'webpack';
+import * as path from 'path';
+import * as loaderUtils from 'loader-utils';
+
+const fabricAsyncLoader: webpack.loader.Loader = (content: string) => {
+  return content;
+};
+
+fabricAsyncLoader.pitch = (remainingRequest: string, precedingRequest: string) => {
+  const moduleRequest = `!!${remainingRequest}`;
+  const normalizedRequest = loaderUtils.stringifyRequest(this, moduleRequest);
+  const moduleName = path.basename(normalizedRequest).replace(/\..*$/, '');
+  return [
+    "import Loadable from 'react-loadable';",
+    `export const ${moduleName} = Loadable({`,
+    `  loader: async() => (await import(${loaderUtils.stringifyRequest(this, moduleRequest)})).${moduleName},`,
+    `  loading: () => null`,
+    `});`
+  ].join('');
+};
+
+export = fabricAsyncLoader;
```

Comment:
Please add documentation to this file and also just use es6 syntax please :) (be consistent with rest of repo)

Reply:
I can't use es6 syntax. Loaders need to be exported this way. I can add a little bit of doc but this loader doesn't do much yet (if ever?)

Reply:
doc added

Reply:
I had thought though that the .ts file is not used as the loader immediately, and in the compilation to js, this will turn into commonjs syntax and thereby become compatible...?